backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java

```diff
@@ -46,7 +46,7 @@
     @ConfigProperty(name = "rbac.retry.max-attempts", defaultValue = "3")
     long maxRetryAttempts;
 
-    @ConfigProperty(name = "rbac.retry.back-off.initial-value", defaultValue = "0.1S")
+    @ConfigProperty(name = "rbac.retry.back-off.initial-value", defaultValue = "0.2S")
```

Comment:
I moved this one a bit to increase the wait to 0.2s, 0.4s and 0.8s between tries.